.eslintrc.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
   extends: ['@energyweb'],
+  rules: {
+    "@typescript-eslint/no-shadow": ["warn", {"ignoreFunctionTypeParameterNameValueShadow": true}],
```

Comment:
@jrhender @JGiter I added this ignoreFunctionTypeParameterNameValueShadow so that you didn't get a warning when you had a function param with a name that is repeated, so we can call parameters what they actually are rather than come up with different words for the same thing every time we have a method. Does this sound okay to you?